types/scripts/build-types.ts

```diff
@@ -11,27 +11,9 @@ import { getFilePath } from "../src/utils";
 
 const OUTPUT_PATH = getFilePath("types/definitions");
 const ENTRYPOINTS = [
-  { compatDate: "2021-01-01", name: "oldest" },
-  // https://developers.cloudflare.com/workers/platform/compatibility-dates/#formdata-parsing-supports-file
-  { compatDate: "2021-11-03" },
-  // https://developers.cloudflare.com/workers/platform/compatibility-dates/#settersgetters-on-api-object-prototypes
-  { compatDate: "2022-01-31" },
-  // https://developers.cloudflare.com/workers/platform/compatibility-dates/#global-navigator
-  { compatDate: "2022-03-21" },
-  // https://developers.cloudflare.com/workers/platform/compatibility-dates/#r2-bucket-list-respects-the-include-option
-  { compatDate: "2022-08-04" },
-  // https://developers.cloudflare.com/workers/platform/compatibility-dates/#new-url-parser-implementation
-  { compatDate: "2022-10-31" },
-  // https://developers.cloudflare.com/workers/platform/compatibility-dates/#streams-constructors
-  // https://developers.cloudflare.com/workers/platform/compatibility-dates/#compliant-transformstream-constructor
-  { compatDate: "2022-11-30" },
-  // https://github.com/cloudflare/workerd/blob/fcb6f33d10c71975cb2ce68dbf1924a1eeadbd8a/src/workerd/io/compatibility-date.capnp#L275-L280 (http_headers_getsetcookie)
-  { compatDate: "2023-03-01" },
-  // https://github.com/cloudflare/workerd/blob/fcb6f33d10c71975cb2ce68dbf1924a1eeadbd8a/src/workerd/io/compatibility-date.capnp#L307-L312 (urlsearchparams_delete_has_value_arg)
-  { compatDate: "2023-07-01" },
-  // Latest compatibility date with experimental features
-  { compatDate: "2999-12-31", name: "latest" },
-  // Latest compatibility date with experimental features
+  // Latest compatibility date
+  { compatDate: "2999-12-31", name: "" },
```

Comment:
Does this result in a separate export to `experimental` below or are they the same?